apps/studio/src/lib/db/authentication/azure.ts

```diff
@@ -153,6 +171,31 @@ export class AzureAuthService {
     }
   }
 
+  private async getAzureCLIToken(): Promise<AuthConfig | null> {
+    return new Promise((resolve, reject) => {
+      const command = "az account get-access-token --resource https://ossrdbms-aad.database.windows.net --output json";
```

Comment:
Beekeeper will need to specifically resolve the az command? Like with backup/restore we should auto-detect this, but if we can't find it we'll need the user to provide the path to it.

We should be really explicit in the form about the fact it is using the `az` client to authenticate.

Also -- this doesn't seem to use the clientSecret or the msiendpoint?